addons/hr_timesheet/static/src/components/time_hour_field/time_hour_field.js

```diff
@@ -0,0 +1,19 @@
+import { registry } from "@web/core/registry";
+import { FloatTimeField } from "@web/views/fields/float_time/float_time_field";
+
+export class TimeHourField extends FloatTimeField {
+    get formattedValue() {
+        const unitAmount = super.formattedValue;
+        const [hours, minutes] = unitAmount.split(":");
+        let durationDisplay = hours + 'h';
+        if (minutes !== '00') {
+            durationDisplay += minutes + 'm';
+        }
```

Comment:
Sorry for having you change this again but I had forgotten to take into account translations when I gave you the previous suggestion. 
Since we now have something language specific in our duration format, we need to take into account how that might change when odoo is used in other languages. 
That's why we need to use something along those lines:
```suggestion
        if (minutes !== '00') {
            return _t("%(hours)sh%(minutes)sm", { hours, minutes });
        }
        return _t("%(hours)sh", hours);
```